common/grpctracing/interceptor.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package grpctracing
+
+import (
+	"go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
+	"google.golang.org/grpc"
+)
+
+func UnaryServerInterceptor() grpc.UnaryServerInterceptor {
+	return otelgrpc.UnaryServerInterceptor()
```

Comment:
I thought some more and I think we should add an option to opt-in in the configuration (`core.yaml` or `orderer.yaml`). 

I am not sure it's a good idea to load the `otelgrpc` by default, all the time. 
It's true that you configure it with environment variables but let's minimize the potential surface area risk.